besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2352,16 +2355,46 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
   }
 
   private GenesisConfigFile getGenesisConfigFile() {
-    return GenesisConfigFile.fromConfig(genesisConfig());
+    return GenesisConfigFile.fromConfigWithoutAccounts(genesisConfig());
   }
 
+  private String genesisConfigString = "";
+
   private String genesisConfig() {
```

Comment:
now this method does not only return the genesis config, so the name is no more descriptive of its behavior

Reply:
Do you mean the method "genesisConfig()"?